torch/nn/modules/lazy.py

```diff
@@ -256,8 +280,10 @@ def _infer_parameters(self: _LazyProtocol, module, args, kwargs=None):
         delattr(module, '_load_hook')
         if module.cls_to_become is not None:
             module.__class__ = module.cls_to_become
-
+    # fmt: on
```

Comment:
`torchrec` requires this to be exactly the same as the one in `torchrec.modules.lazy_extension.LazyModuleExtensionMixin._infer_parameters`

https://github.com/pytorch/torchrec/blob/eca606d1ccb66ccec823254675a1eff5853270b1/torchrec/modules/lazy_extension.py#L159-L182

https://github.com/pytorch/torchrec/blob/eca606d1ccb66ccec823254675a1eff5853270b1/torchrec/modules/tests/test_lazy_extension.py#L57-L83